torchvision/models/detection/transform.py

```diff
@@ -88,17 +116,23 @@ def resize(self, image, target):
         return image, target
 
     def batch_images(self, images, size_divisible=32):
+        # type: (List[Tensor], int)
         # concatenate
-        max_size = tuple(max(s) for s in zip(*[img.shape for img in images]))
-
-        stride = size_divisible
+        # TODO: Fix this
+        # max_size = tuple(max(s) for s in zip(*[img.shape for img in images]))
+        max_size = torch.jit.annotate(List[int], [])
+        for img in images:
+            # for s in img:
+            max_size.append(self.max_DELETEME(img))
```

Comment:
This is not doing the same thing as before.
I believe the correct replacement would be something like
```python
for dim in range(images[0].dim()):
    t = 0
    for img in images:
        t = self.max_DELETEME([t, img.size(dim)])
    max_size.append(t)
```